Corvus.Extensions.Samples/ListExtensionsSample.dib

```diff
@@ -0,0 +1,53 @@
+#!markdown
+
+## ListExtensions - Sample Notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+#!markdown
+
+Add using statement for `Corvus.Extensions` namespace
+
+#!csharp
+
+using Corvus.Extensions;
+
+#!markdown
+
+### `RemoveAll()`
+
+Example using the `RemoveAll()` extension method. This removes all items from a list that match a predicate.
+
+#!markdown
+
+### Examples
+
+#!markdown
+
+Example 1 using a list of `int`
+
+#!csharp
+
+var list = new List<int> { 1, 2, 3, 4, 5, 6 };
+list.RemoveAll(i => i < 3);
+
```

Comment:
You're not outputting the list after the statement, so we don't see the result when the cell is run

Reply:
Fixed ✅